electron/build/template-package.json

```diff
@@ -148,6 +148,12 @@
     "vscode-arduino-tools": "https://downloads.arduino.cc/vscode-arduino-tools/vscode-arduino-tools-0.0.2-beta.2.vsix",
     "vscode-builtin-json": "https://open-vsx.org/api/vscode/json/1.46.1/file/vscode.json-1.46.1.vsix",
     "vscode-builtin-json-language-features": "https://open-vsx.org/api/vscode/json-language-features/1.46.1/file/vscode.json-language-features-1.46.1.vsix",
-    "cortex-debug": "https://open-vsx.org/api/marus25/cortex-debug/0.3.10/file/marus25.cortex-debug-0.3.10.vsix"
+    "cortex-debug": "https://open-vsx.org/api/marus25/cortex-debug/0.3.10/file/marus25.cortex-debug-0.3.10.vsix",
+    "vscode-language-pack-nl": "https://open-vsx.org/api/MS-CEINTL/vscode-language-pack-nl/1.48.3/file/MS-CEINTL.vscode-language-pack-nl-1.48.3.vsix",
+    "vscode-language-pack-fr": "https://open-vsx.org/api/MS-CEINTL/vscode-language-pack-fr/1.66.1/file/MS-CEINTL.vscode-language-pack-fr-1.66.1.vsix",
+    "vscode-language-pack-zh-hant": "https://open-vsx.org/api/MS-CEINTL/vscode-language-pack-zh-hant/1.66.1/file/MS-CEINTL.vscode-language-pack-zh-hant-1.66.1.vsix",
```

Comment:
Is this the right one? There are two localization initiatives for Arduino IDE related to Chinese languages:

- "**Chinese**": Currently at 100% localization coverage: https://github.com/arduino/arduino-ide/blob/install-language-packs/i18n/zh.json
- "**Chinese (Taiwan)**": Currently at 2.8% coverage: https://github.com/arduino/arduino-ide/blob/install-language-packs/i18n/zh_TW.json

[The language code used by the "`MS-CEINTL.vscode-language-pack-zh-hant`" language pack is `zh-tw`](https://github.com/microsoft/vscode-loc/blob/main/i18n/vscode-language-pack-zh-hant/package.json#L22)

Reply:
👍  changed it with vscode-language-pack-zh-hans, thanks!!